### PR TITLE
update contributing.md

### PR DESCRIPTION
**Pull request by fredericrous**
I feel the (future) contributor could be guided a bit more on how to build and run the project.
This PR's goal is to give insight on how to build and run the solution on users' local env

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -31,6 +31,13 @@ And finally install [golangci-lint](https://github.com/golangci/golangci-lint#in
 
 You might want to use an editor such as [Visual Studio Code](https://code.visualstudio.com/download) with the [Go extension](https://code.visualstudio.com/docs/languages/go). Working settings are already in [.vscode/settings.json](../.vscode/settings.json).
 
+## Build and Run
+
+```sh
+go build -o app cmd/updater/main.go
```
Comment (qdm12) [inline docs/contributing.md:37]: Maybe make that a `go run` instead? 😉 Also I don't think you need to specify LISTENING_PORT as it defaults to `8000`? If it doesn't it's a bug!

**Comment by qdm12** [comment]
Thanks a bunch! Feel free to open a discussion on Github if you want to chat if you want to contribute in Go 😉

**Comment by qdm12** [comment]
Never mind, didn't see your other PR! Will review that this evening! And I haven't throughly tested the CI for PR forks, I'll fix it up if it doesn't pass. Thanks!

File: docs/contributing.md (hunk 1 above)
Comment (fredericrous) [reply docs/contributing.md]: you caught me 😆 I am not very familiar with Go, I didn't know about the run command. I updated the file, I am also gonna create an issue for LISTENING_PORT, it does not default to 8000

Comment (fredericrous) [reply docs/contributing.md]: actually if I run `go run cmd/updater/main.go`

I get the following error:

```sh
2021-01-25T21:33:56.240Z        ERROR   open /tmp/go-build849192135/b001/exe/data/config.json: no such file or directory
```

I am using vs code workspaces

Comment (qdm12) [reply docs/contributing.md:37]: ah yes good point, let's leave it as you have it then 👍

File: docs/contributing.md [hunk 2]
```diff
@@ -31,6 +31,13 @@ And finally install [golangci-lint](https://github.com/golangci/golangci-lint#in
...
+LISTENING_PORT=8000 ./app
```
Comment (qdm12) [inline docs/contributing.md:38]:
```suggestion
./app
```

**Review by qdm12 — APPROVED** [review]

**Pull request merged**